pkg/decision/entities.go

```diff
@@ -56,6 +57,7 @@ type FeatureDecision struct {
 	Source     Source
 	Experiment entities.Experiment
 	Variation  *entities.Variation
+	Variable   *entities.Variable
```

Comment:
Like mentioned above, I don't think you need to pass it in the decision. It's enought that it's in the context

Reply:
addressed